torch/ao/quantization/experimental/observer.py

```diff
@@ -4,44 +4,135 @@
 """
 
 import torch
+import itertools
 from torch.ao.quantization.observer import ObserverBase
 from typing import Tuple
 
 class NonUniformQuantizationObserverBase(ObserverBase):
-    quant_min = None
-    quant_max = None
+    min_val: torch.tensor([])
```

Comment:
i think this is just torch.Tensor

Reply:
torch.Tensor will be deprecated in the future. torch.tensor's constructor is a bit more aligned with numpy array than torch.Tensor. i think this lint error is talking about something else